Dockerfile

```diff
@@ -1,24 +1,25 @@
 FROM php:7.0.8-alpine
 
-COPY ./src /usr/src/gush/src
-COPY ./gush /usr/src/gush/gush
-COPY ./composer.json /usr/src/gush/composer.json
-
-WORKDIR /usr/src/gush
-
 RUN set -xe \
     && apk add --no-cache \
     git \
     openssh-client
 
 RUN curl -s https://getcomposer.org/installer | php \
     && chmod +x composer.phar \
-    && COMPOSER_ALLOW_SUPERUSER=1 php composer.phar install --prefer-dist --optimize-autoloader --no-interaction --no-dev \
-    && rm composer.phar \
-    && rm composer.json \
-    && rm composer.lock
+    && mv composer.phar /usr/bin/composer
 
 RUN mkdir /root/project
 WORKDIR /root/project
 
+COPY ./src /usr/src/gush/src
+COPY ./gush /usr/src/gush/gush
+COPY ./composer.json /usr/src/gush/composer.json
+
+WORKDIR /usr/src/gush
```

Comment:
Last `WORKDIR` must be `/root/project/`, since there should be the mounted repo from which gush should be executed (see #613).